src/Expensify.tsx

```diff
@@ -206,6 +207,10 @@ function Expensify({
         // eslint-disable-next-line react-hooks/exhaustive-deps -- we don't want this effect to run again
     }, []);
 
+    useEffect(() => {
```

Comment:
Can we add a comment that we have to override this setting to allow playing music in silent mode and align with behavior on android and other platforms